poc_iot_injector/src/receipt_txn.rs

```diff
@@ -0,0 +1,244 @@
+use crate::{keypair::Keypair, Error, Result};
+use helium_crypto::Sign;
+use helium_proto::{
+    services::poc_lora::{
+        LoraBeaconReportReqV1, LoraValidBeaconReportV1, LoraValidPocV1, LoraValidWitnessReportV1,
+        LoraWitnessReportReqV1,
+    },
+    BlockchainPocPathElementV1, BlockchainPocReceiptV1, BlockchainPocWitnessV1,
+    BlockchainTxnPocReceiptsV2, Message,
+};
+use rust_decimal::{prelude::ToPrimitive, Decimal, MathematicalOps};
+use rust_decimal_macros::dec;
+use sha2::{Digest, Sha256};
+
+// TODO: These should ideally be coming in from some configuration service
+const WITNESS_REDUNDANCY: u32 = 4;
+const POC_REWARD_DECAY_RATE: Decimal = dec!(0.8);
+const HIP15_TX_REWARD_UNIT_CAP: Decimal = dec!(2.0);
+
+// TODO: This should be done better?
+const MHZTOHZ: u32 = 1000000;
```

Comment:
perhaps a macro?

Reply:
use a function .. I think you have one already.. have the multiplier there and remove this const

Reply:
Updated 49d15fd